CONTRIBUTING.md

```diff
@@ -50,17 +50,84 @@ type checking. Any code changes that you contribute should pass these checks.
 The easiest way to get set up for development is to run the following script:
 
 ```bash
+# first fork the repo on github
+
+#then clone your fork
+git clone https://github.com/<your-github-username>/gluon-ts.git
+
+# check if condo is up to date
+Conda update conda
+
+# create virtual env (can change python version if you use newer/older
+# see if any breaking changes
+conda create -n gluon-ts python=3.6 anaconda
+
+# after aeons of dependencies installing
+conda activate gluon-ts
+
+#confirm python version
+Python -V
+
+#enter the cloned repo
 ./dev_setup.sh
+
+#NOTE: you may get some setup lines that are red in stdout
+#In my case it looks like those were ok, on re-running ./dev_setup.sh had no red #stdout lines
+
+# now setup the main repo as upstream remote
+git remote add upstream https://github.com/awslabs/gluon-ts.git
 ```
 
 This will install all the requirements and also install a git hook that runs
-all code checks when you commit.
+all code checks when you commit. This also separates your workflow so that
+pull-requests go through your forked repo and then into upstream if approved.
 
 
 ## Finding contributions to work on
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/awslabs/gluon-ts/labels/help%20wanted) issues is a great place to start.
 
 
+## Reviewing code
```

Comment:
I think this section could be avoided: people looking into reviewing code probably have their preferred workflow and habits.

Reply:
I can remove this section but I'm not sure how else someone would run tests locally without having a local checkout of the code in the PR?